tests/bincode.rs

```diff
@@ -22,7 +22,7 @@ proptest! {
                 assert_eq!(pk_bytes_from, pk_bytes_bincode);
 
                 // Check 2: bincode encoding should match original bytes.
-                let bytes_bincode = bincode::serialize(&sk_from).unwrap();
+                let bytes_bincode = bincode::serialize(&sk_from)?;
```

Comment:
I suggest reverting all of the changes in tests. Tests should normally fail as quickly as possible if something is unexpected.